### PR TITLE
Show questions from backend

### PR DESCRIPTION
**Pull request by nicholasdalhaug**

### CODE REVIEW

File: frontend/src/components/QuestionAndAnswer/QuestionAndAnswerFormWithApi.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    createDate: "",
+    answeredBy: emptyParticipant
+}
+
+interface QuestionAndAnswerFormWithApiProps {
+    questionNumber: number
+    question: Question
+}
+
+const QuestionAndAnswerFormWithApi = ({questionNumber, question}: QuestionAndAnswerFormWithApiProps) => {
```
Comment (tmals-equi) [inline frontend/src/components/QuestionAndAnswer/QuestionAndAnswerFormWithApi.tsx:64]: Not sure about this name, could we change it to something like QuestionAndAnswerFormWithData?

**Review by tmals-equi — APPROVED** [review]
Looks good apart from one minor question

File: frontend/src/components/QuestionAndAnswer/QuestionAndAnswerFormWithApi.tsx (hunk 1 above)
Comment (nicholasdalhaug) [reply frontend/src/components/QuestionAndAnswer/QuestionAndAnswerFormWithApi.tsx:64]: Doesn't that make it sound like it needs more data as input, when the case is that it needs less? I was also thinking that this whole thing should not be necessary and that we should get big queries to work correctly with the cache. What do you think about that?

Comment (tmals-equi) [reply frontend/src/components/QuestionAndAnswer/QuestionAndAnswerFormWithApi.tsx:64]: Yeah, I agree. But maybe that could be a separate issue?

Comment (nicholasdalhaug) [reply frontend/src/components/QuestionAndAnswer/QuestionAndAnswerFormWithApi.tsx:64]: You are probably right.

**Pull request merged**